.github/workflows/specs.yml

```diff
@@ -2,9 +2,8 @@ name: "Test specs"
 
 on:
   pull_request:
-  branches:
-    - master
-
```

Comment:
I think here is "push:" missing

Reply:
Branches is now attached to `pull_request` so it should be fine. However we probably want these tests to also run on dependent PR (that are not necessarily based on `master`), so I imagine the thing to do is to remove the branches filter entirely.